### ISSUE TITLE
Instead of integrating neume-network/strategies as a git submodule, might it make more sense to include it in core as a separate npm package?

### ISSUE THREAD

**Issue by TimDaub** [neume-network/strategies#52]

**Comment by il3ven** [comment]
An argument in favour of npm package is that it would enable consistency. iirc message-schema is being imported as a module.

**Comment by TimDaub** [comment]
I agree. Another argument is that between core and strategies, we want to create a social contract that e.g. some semantically versioned interfaces have to be enabled. At one stage, e.g. it may need to be the core's responsibility to invoke strategy lifecycle methods.
